api/v1/kusk.yaml

```diff
@@ -118,7 +118,7 @@ spec:
             scriptContent:
               description: script metadata content
               type: string
-            scriptName:
+            testName:
               description: unique script name (CRD Script name)
```

Comment:
This file contains a lot of Script terms. And it's not synchronized with latest models. We use simply `name` field for example